<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupWindow</class>
 <widget class="QWidget" name="GroupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Зберегти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Скасувати</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>55</number>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Назва&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="specialLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Спеціальність&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearStartLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Рік початку навчання&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearEndLabel">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>340</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Рік закінчення навчання&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="curatorLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Куратор&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="headManLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Староста&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="curatorComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Оберіть куратора</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>rew</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>wrewr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>wer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>wer</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="headManComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Оберіть старосту</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="startStudySpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>135</width>
     <height>21</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>2000</number>
   </property>
   <property name="maximum">
    <number>2100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="endStudySpinBox">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>360</y>
     <width>135</width>
     <height>21</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>2001</number>
   </property>
   <property name="maximum">
    <number>2100</number>
   </property>
  </widget>
  <widget class="QLabel" name="mainImage">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="specialComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Оберіть спеціальність</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>051 - Економіка</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>121 - Інеженерія программного забезпечення</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>131 - Прикладна механіка</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="okLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>125</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%status_label%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
